statsmodels/tools/tests/test_sequences.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import numpy as np
+import numpy.testing as npt
+from statsmodels.tools import sequences
+
+
+def test_discrepancy():
+    space_1 = [[1, 3], [2, 6], [3, 2], [4, 5], [5, 1], [6, 4]]
+    space_2 = [[1, 5], [2, 4], [3, 3], [4, 2], [5, 1], [6, 6]]
+
+    corners = np.array([[0.5, 0.5], [6.5, 6.5]])
+
+    # From Fang et al. Design and modeling for computer experiments, 2006
+    assert sequences.discrepancy(space_1, corners) == pytest.approx(0.0081, abs=0.0001)
+    assert sequences.discrepancy(space_2, corners) == pytest.approx(0.0105, abs=0.0001)
```

Comment:
I think you can use numpy's assert_allclose instead

Reply:
Done